test/services/database/post_repository_test.dart

```diff
@@ -48,22 +46,12 @@ void main() {
       );
     });
 
-    final post = PostFirestore(
-      id: const PostIdFirestore(value: "post_id"),
-      location: const PostLocationFirestore(
-        geoPoint: GeoPoint(40, 20),
-        geohash: "afed",
-      ),
-      data: PostData(
-        ownerId: const UserIdFirestore(value: "owner_id"),
-        title: "post_tiltle",
-        description: "description",
-        publicationTime: Timestamp.fromMillisecondsSinceEpoch(4564654),
-        voteScore: 12,
-      ),
+    final post = MockPostFirestore.createPostAt(
+      MockPostFirestore.generatePostData(1).first,
+      userPosition2,
     );
```

Comment:
I think that we can maybe abstract this method, as I saw duplication of post creation at a certain location without using the actual post generated.

So we may want a function that given a location create a post, without needing to provide `PostData`. This function could be named: `generatePostAt` in the `MockPostFirestore`

Alternative representation (line 90, 80, 104, 118): `final postData = MockPostFirestore.generatePostData(1).first;
      final expectedPost =
          MockPostFirestore.createPostAt(postData, farAwayPostPosition);`